.env.example

```
@@ -0,0 +1,98 @@
+NEXT_PUBLIC__PROJECT_ID="your_firebase_project_id_here"
+NEXT_PUBLIC__STORAGE_BUCKET="your_firebase_storage_bucket_here"
+NEXT_PUBLIC__MESSAGING_SENDER_ID="your_firebase_messaging_sender_id_here"
+NEXT_PUBLIC__APP_ID="your_firebase_app_id_here"
+
+# ============================================
+# BACKEND API CONFIGURATION
+# ============================================
+# Set this to your backend API URL
+# For local development: http://localhost:3001
+# For staging: https://devapi.dehix.org/
+NEXT_PUBLIC__BASE_URL="http://localhost:8080/"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)